vspace/src/main/webapp/WEB-INF/spring/root-context.xml

```diff
@@ -1,43 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<beans xmlns="http://www.springframework.org/schema/beans"
-    xmlns:se="http://www.springframework.org/schema/security"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xmlns:mvc="http://www.springframework.org/schema/mvc"
-    xmlns:context="http://www.springframework.org/schema/context"
-    xmlns:util="http://www.springframework.org/schema/util"
-    xmlns:aop="http://www.springframework.org/schema/aop"
-    xmlns:task="http://www.springframework.org/schema/task"
-    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
-	http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc.xsd
-	http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
-	http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
-	http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security-5.3.xsd
-	http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-3.2.xsd
-	http://www.springframework.org/schema/task http://www.springframework.org/schema/task/spring-task-3.0.xsd">
-
-    <mvc:annotation-driven />
-    <context:component-scan
-        base-package="edu.asu.diging.vspace, edu.asu.diging.simpleusers" />
-    <bean id="messageSource"
-        class="org.springframework.context.support.ReloadableResourceBundleMessageSource">
-        <property name="basename"
-            value="classpath:locale/messages" />
-        <property name="fallbackToSystemLocale" value="false" />
-    </bean>
-
-    <aop:aspectj-autoproxy
-        proxy-target-class="true" />
-
-    <bean
-        class="org.springframework.context.support.PropertySourcesPlaceholderConfigurer">
-        <property name="propertiesArray">
-            <array>
-                <ref bean="configFile" />
-            </array>
-        </property>
-    </bean>
-
-    <util:properties id="configFile"
-        location="classpath:config.properties" />
```

Comment:
all these beans need to be put into the javaconfig

Reply:
or at least the messagesource one if you have tested everything and the default config file is actually not used.